# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/style/moz.build
# Commit: 93b7e171c74a
# Full Hash: 93b7e171c74a6aa88d98979009eb0e299e2b61a3
# Author: Andrew Osmond <aosmond@mozilla.com>
# Date: 2022-06-29 03:44:30
# Regressor Bug: 1771493
# File Overlap Count: 1
# Description:
#   Bug 1771493 - Part 1. Split FontFace(Set) into DOM facing and implementation classes. r=emilio
#   
#   This patch aims to split the implementation details of FontFace(Set)
#   into FontFace(Set)Impl classes. The Impl variants have threadsafe
#   refcounting and are intended to be the basis for worker support for
# ==============================================================================

diff -r c45dacd63805 -r 93b7e171c74a layout/style/moz.build
--- a/layout/style/moz.build	Tue Jun 28 14:12:34 2022 +0000
+++ b/layout/style/moz.build	Tue Jun 28 14:27:20 2022 +0000
@@ -140,7 +140,9 @@
     "CSSSupportsRule.h",
     "CSSValue.h",
     "FontFace.h",
+    "FontFaceImpl.h",
     "FontFaceSet.h",
+    "FontFaceSetImpl.h",
     "FontFaceSetIterator.h",
     "MediaList.h",
     "MediaQueryList.h",
@@ -188,7 +190,9 @@
     "DocumentStyleRootIterator.cpp",
     "ErrorReporter.cpp",
     "FontFace.cpp",
+    "FontFaceImpl.cpp",
     "FontFaceSet.cpp",
+    "FontFaceSetImpl.cpp",
     "FontFaceSetIterator.cpp",
     "FontPreloader.cpp",
     "GeckoBindings.cpp",